mock.js

```diff
@@ -0,0 +1,5 @@
+import mockAsyncStorage from '@react-native-async-storage/async-storage/jest/async-storage-mock';
+
+jest.mock('expo-constants', () => ({ manifest: { version: '1.0.0' } }));
+jest.mock('@react-native-async-storage/async-storage', () => mockAsyncStorage);
+jest.mock('react-native/Libraries/Animated/NativeAnimatedHelper');
```

Comment:
Est-ce que ce mock est utile?

Reply:
Pourquoi il ne le serait pas ? Il était dans le fichier de test avant, je l'ai juste changé de place. je ne sais pas trop à quoi il sert

Reply:
Je posais la question car de mon coté, les tests passent sans ce mock. 
J'ai l'impression que ce mock ne sert a rien pour nos tests, il pourrait servir dans le cas ou on teste la navigation entre plusieurs pages mais ce n'est pas le cas ici

Reply:
C'est supprimé !

Reply:
Quand je le supprime j'ai un warning :  `Animated: `useNativeDriver` is not supported because the native animated module is missing. Falling back to JS-based animation. To resolve this, add `RCTAnimation` module to this app, or remove `useNativeDriver`. Make sure to run `pod install` first. Read more about autolinking: https://github.com/react-native-community/cli/blob/master/docs/autolinking.md`

Tu ne l'as pas ?

Reply:
Si pardon je ne l'avais pas vu !